discord/emoji.py

```diff
@@ -306,6 +306,9 @@ def is_usable(self):
             An error occurred editing the emoji.
         """
 
+        name = fields.get('name', self.name)
```

Comment:
```suggestion
        name = name or self.name
```

Following the signature change.